CODEOWNERS

```
@@ -11,3 +11,4 @@ go-libs                    @nfx
 ip_access_list_analyzer    @alexott
 metascan                   @nfx
 runtime-packages           @nfx
+uc-catalog-cloning         @esiol-db, @vasco-lopes
```

```suggestion
uc-catalog-cloning         @esiol-db @vasco-lopes
```
this line was an error